lib/components/settings/config/cubit/config.dart

```diff
@@ -20,13 +21,16 @@ class Config extends Equatable {
   /// Whether to automatically hide the home tab bar when swiping up / down.
   final bool hideHomeTabBar;
 
+  final bool showAbsoluteTime;
```

Comment:
Because `Config` uses [equatable](https://pub.dev/packages/equatable),  it uses the `props` list to handle its equality (to override the object's `==` and `hashCode`).

Because you haven't added the new field to the `props` list, the cubit thinks the config with `showAbsoluteTime` `true or `false` is the same and thus doesn't notify its listeners when only that field changes.

Long story short, add `showAbsoluteTime` in the `props` list in `Config` and the switch will update as expected.

Reply:
Ohhhh, I forgot to do this!!!! It's working, thank you!